src/Coordination/KeeperDispatcher.cpp

```diff
@@ -158,6 +170,7 @@ void KeeperDispatcher::requestThread()
                     {
                         addErrorResponses(current_batch, Coordination::Error::ZCONNECTIONLOSS);
                         current_batch.clear();
+                        current_batch_bytes_size = 0;
```

Comment:
why do we reset it here?

Reply:
it's reset so that it remains consistent after current_batch.clear();